sim-lib/src/lib.rs

```diff
@@ -326,7 +367,7 @@ impl Simulation {
         let mut success = true;
         while let Some(res) = tasks.join_next().await {
             if let Err(e) = res {
-                log::error!("Task exited with error: {e}.");
+                log::error!("Task exited with error: {}.", e);
```

Comment:
Does this need to change?

Reply:
Not really. The syntax Just didn't look right to me.

Reply:
I think this is pretty canonical rust so let's leave as is?